server/bin/timely-standalone.sh

```diff
@@ -25,12 +25,15 @@ if [[ -e ${NATIVE_DIR} ]]; then
 fi
 mkdir -p ${NATIVE_DIR}
 
-$JAVA_HOME/bin/jar xf ${LIB_DIR}/netty-tcnative*.jar META-INF/native/libnetty-tcnative.so
+pushd ${BASE_DIR}/bin
+echo "$JAVA_HOME/bin/jar xvf ${LIB_DIR}/netty-tcnative*.jar META-INF/native/libnetty-tcnative.so"
+$JAVA_HOME/bin/jar xvf ${LIB_DIR}/netty-tcnative*.jar META-INF/native/libnetty-tcnative.so
```

Comment:
I think @brianloss mentioned that we should extract `META-INF/native/libnetty-tcnative` so that it works on a mac.